README.md

```diff
@@ -29,6 +29,14 @@ To access the mongo shell, run:
 make mongo-shell
 ```
 
+### State reset
+
+During development, you may need to get an application state, where no data is published.
+
+One way is to modify `config/development.js`, and change the `dbName` to a new one.
+
+> **Note**: an alternative way is to use the mongo-shell and drop the database.
+
```

Comment:
No, we supply two make commands for that:

`make clear-publication`: just clear the published data but keep your uploaded dataset and your model
`make clear-database`: clear the whole database